src/panels/lovelace/custom-card-helpers.ts

```diff
@@ -3,3 +3,4 @@ export { createCardElement } from "./create-element/create-card-element";
 export { createHeaderFooterElement } from "./create-element/create-header-footer-element";
 export { createHuiElement } from "./create-element/create-hui-element";
 export { createRowElement } from "./create-element/create-row-element";
+export { importMoreInfoControl } from "../../dialogs/more-info/state_more_info_control";
```

Comment:
Should it be a creator rather than a loader to follow the pattern? This feels a bit like relying on side effects.

Reply:
There is no point in a creator here, as it doesn't have a `setConfig` function or anything. It is way easier to just use it in a template instead of having to update the elements properties manual all the time? 🤷‍♂️

Reply:
Yeah, I can definitely see why, and I actually think it's reasonable... but the side effect thing smells.

Reply:
That we allow this smells, we just don't have a better alternative atm...